www/_includes/layouts/menu.njk

```diff
@@ -21,28 +21,28 @@
       <span class="menu-section-header">Getting Started</span>
       <ol>
         <li>
-          <a href="/#">Install Snowpack</a>
+          <a href="/tutorials/getting-started">Tutorial</a>
```

Comment:
same as below, can we make this change:

1. Convert the current "getting-started" guide into a "quick-start" guide.
2. Create a new page called "Install Snowpack". This can be pretty short, but people like to click it for obvious "how to install" instructions. I'm happy to write out a stub for this of what I'm thinking, but just want to keep the sidebar item for now.

Reply:
#1 would require a bit of a rewrite since it's pretty in depth right now. Don't you think #2 is covered by the home page now?

Reply:
yea, definitely not looking for a full rewrite now. Okay to keep as is if those requests don't make sense, and I can take a stab at it when you pass it back to me.